src_features/signMessageEIP712/ui_logic.h

```diff
@@ -0,0 +1,47 @@
+#ifndef UI_LOGIC_712_H_
+#define UI_LOGIC_712_H_
+
+#ifdef HAVE_EIP712_FULL_SUPPORT
+
+#include <stdint.h>
+#include "ux.h"
+
+#define UI_712_FIELD_SHOWN         (1 << 0)
+#define UI_712_FIELD_NAME_PROVIDED (1 << 1)
+
+typedef enum { EIP712_FILTERING_BASIC, EIP712_FILTERING_FULL } e_eip712_filtering_mode;
```

Comment:
single line enum are not readable IMO

Reply:
Will look into choosing a better clang-format configuration in a future PR. +1